File: BMotionServices/Parse/ResponseOrders.cs
using BMotionServices.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace BMotionServices.Parse
{
    public class ResponseOrders
    {
        public string status { get; set; }
        public string message { get; set; }
        public Orders Data { get; set; }
    }
}

File: app/src/main/java/com/stratone/bmotion/response/ResponseOrders.java
package com.stratone.bmotion.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import com.stratone.bmotion.model.Orders;

public class ResponseOrders {
    @SerializedName("status")
    @Expose
    private String status;
    @SerializedName("message")
    @Expose
    private String message;
    @SerializedName("Data")
    @Expose
    private Orders data;

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public Orders getData() {
        return data;
    }

    public void setData(Orders data) {
        this.data = data;
    }
}
